test/fuzz/lua/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
+
+set(LUA_PATH "${CMAKE_CURRENT_SOURCE_DIR}/?.lua\;\;")
+
+set(TEST_SUITE_NAME "fuzzing-lua")
+
+message(STATUS "Add test suite ${TEST_SUITE_NAME}")
+
+# XXX: The call produces both test and target
+# <${TEST_SUITE_NAME}-deps> as a side effect.
+_add_test_suite_target(${TEST_SUITE_NAME}
+  LABELS ${TEST_SUITE_NAME}
+  DEPENDS tarantool
+)
+
+set(test_path "test/fuzz/lua")
+add_test(NAME ${test_path}/test_engine.lua-memtx
+  COMMAND ${TARANTOOL_BIN}
+          ${CMAKE_CURRENT_SOURCE_DIR}/test_engine.lua
+          --test_duration 60
+          --workers 500
+          --seed ${RANDOM_SEED}
+          --engine memtx
+)
+add_test(NAME ${test_path}/test_engine.lua-vinyl
+  COMMAND ${TARANTOOL_BIN}
+          ${CMAKE_CURRENT_SOURCE_DIR}/test_engine.lua
+          --test_duration 60
+          --workers 500
+          --seed ${RANDOM_SEED}
+          --engine memtx
```

Comment:
Typo: s/memtx/vinyl/
Maybe to use macro to avoid such mistakes?

Reply:
Fixed:

```diff
--- a/test/fuzz/lua/CMakeLists.txt
+++ b/test/fuzz/lua/CMakeLists.txt
@@ -28,7 +28,7 @@ add_test(NAME ${test_path}/test_engine.lua-vinyl
           --test_duration 60
           --workers 500
           --seed ${RANDOM_SEED}
-          --engine memtx
+          --engine vinyl
 )
 set_tests_properties(
   ${test_path}/test_engine.lua-vinyl
```